packages/devextreme/scss/widgets/fluent/tabs/variables/_colors.scss

```diff
@@ -52,37 +82,63 @@ $tabs-tab-icon-color: $tabs-tab-color !default;
 * $name 80. Selected tab icon color
 * $type color
 */
-$tabs-tab-icon-selected-color: $tabs-tab-selected-color !default;
+$tabs-tab-icon-selected-color: $base-accent !default;
 
 /**
 * $name 90. Disabled tab icon color
 * $type color
 */
-$tabs-tab-icon-disabled-color: color.change($tabs-tab-icon-color, $alpha: 0.32) !default;
+$tabs-tab-icon-disabled-color: $base-color-disabled !default;
 $tabs-tab-button-disabled-opacity: 0 !default;
 
 /**
 * $name 100. Hovered tab background color
 * $type color
 */
-$tabs-hovered-tab-bg-color: color.change($base-accent, $alpha: 0.08) !default;
+$tabs-hovered-tab-bg-color: transparent !default;
 
 /**
 * $name 110. Focused tab background color
 * $type color
 */
-$tabs-focused-tab-bg-color: color.change($base-accent, $alpha: 0.12) !default;
+$tabs-focused-tab-bg-color: null !default;
+
+/**
+* $name 111. Hovered tab background color primary
+* $type color
+*/
+$tabs-tab-bg-color-hover-primary: null !default;
+
+/**
+* $name 112. Focused tab background color primary
+* $type color
+*/
+$tabs-tab-bg-color-focus-primary: null !default;
+
+/**
+* $name 112. Active tab background color primary
```

Comment:
ID `112` using twice and I would be prefer not to publish this variables.

Reply:
[[refactor(fluent): Remove extra variable && Remove ways to publish variables](07cb4534d39dda5d2d3359df2bf5fe85dd3cd4ad)]

Reply:
[[refactor(fluent): Remove extra variable && Remove ways to publish variables](154c4f1fffced1986890ef03376011e70927dd60)]